### PR TITLE
Updates for future beta

### PR DESCRIPTION
**Pull request by ciotlosm**
**Description:**
Updates to changelog for Lovelace for future beta

**Pull request in [home-assistant](https://github.com/home-assistant/home-assistant) (if applicable):** home-assistant/home-assistant#<home-assistant PR number goes here>

## Checklist:

- [ ] Branch: Fixes, changes and adjustments should be created against `current`. New documentation for platforms/components and features should go to `next`.
- [ ] The documentation follow the [standards][standards].

[standards]: https://home-assistant.io/developers/documentation/standards/

### CODE REVIEW

File: source/lovelace/changelog.markdown [hunk 1]
```diff
@@ -9,6 +9,24 @@ sharing: true
 footer: true
 ---
 
+## {% linkable_title Changes in 0.74.0b01 %}
```
Comment (c727) [inline source/lovelace/changelog.markdown]: b0

File: source/lovelace/changelog.markdown [hunk 2]
```diff
@@ -9,6 +9,24 @@ sharing: true
...
+- 🔧 Fix language detection for country codes
```
Comment (c727) [inline source/lovelace/changelog.markdown]: not lovelace related

File: source/lovelace/changelog.markdown [hunk 3]
```diff
@@ -9,6 +9,24 @@ sharing: true
...
+
+### Views
+- Add basic support for `badges` like in old view style
+- Custom cards now work with `panel: true`
+
+### Cards
+- `glance` card supports now `toggle` and `turn_on` besides showing more-info dialog
```
Comment (c727) [inline source/lovelace/changelog.markdown]: turn-on

File: source/lovelace/changelog.markdown [hunk 4]
```diff
@@ -9,6 +9,24 @@ sharing: true
...
+- `glance` card supports now to hide `name` or `state`
+- `history-graph` supports override of history names
```
Comment (c727) [inline source/lovelace/changelog.markdown]: entity names

**Review by balloob — APPROVED** [review]

**Pull request merged**